### PR TITLE
Transfer learning 과제입니다..

### PR DESCRIPTION
**Pull request by yeahoon-k**
- 초기학습모델 최종 loss/acc
p1

- 가중치 transfer후 loss/acc
p2

입니다.

정확도는 72%정도입니다.

Transfer 후에 learning rate을 더 낮추어도  바로 trade-off 구간에 봉착하여 val loss가 증가합니다.

TODO :
1. fine tune이 더 필요합니다.
2. Momentum Optimizer로 변환시 momentum의 초기화 관련 error가발생 합니다.

감사합니다.

### CODE REVIEW

**Comment by pai-plznw4me** [comment]
optimizer을 똑같은걸 사용하면 모델에 변함이 없습니다.
그리고 복원한 모델의 loss 가 다른데 코드상에 문제가 있는것 같습니다. 
확인후 다시 올려주세요!
